### PR TITLE
Release 0.0.28

### PR DESCRIPTION
**Pull request by vontell**

### CODE REVIEW

**Review by batu — APPROVED** [review]

**Review by addisonbgross — APPROVED** [review]

File: docs/changelog.mdx [hunk 1]
```diff
@@ -4,6 +4,28 @@ sidebar_label: 'Changelog'
...
+- Bot segments that rely on image sources can now load images from paths vs just raw data
+- Direct login is no longer supported for the SDK; API keys are now required
+- Bot Sequences are now reloaded automatically from disk when played
+- Object detection bot segments can now define a confidence threshold (text queries only)
+
+**Bug Fixes**
+
+- Improved language and explanations within the website
+- Properly handle failing behaviour segments that dont load or timeout
+- Use of computer-vision capabilities now requires authentication
```
Comment (RG-nAmKcAz) [inline docs/changelog.mdx:23]: `now requires authentication` -> is fine, but we should mention that this explicitly requires using API keys / having an API key setup in the SDK

File: docs/changelog.mdx [hunk 2]
```diff
@@ -4,6 +4,28 @@ sidebar_label: 'Changelog'
...
+- Sprite rendering within the SDK has been improved
+- Fixed a bug where the overlay would throw a null exception when opening for the first time
+- Fixed issues where the game would hang when starting
+- Data from the end of a segment is now properly saved
```
Comment (RG-nAmKcAz) [inline docs/changelog.mdx:27]:
```suggestion
- Data for the last segment of a recording is now properly saved
```

File: docs/changelog.mdx [hunk 3]
```diff
@@ -4,6 +4,28 @@ sidebar_label: 'Changelog'
...
+- Object detection bot segments can now define a confidence threshold (text queries only)
+
+**Bug Fixes**
+
+- Improved language and explanations within the website
+- Properly handle failing behaviour segments that dont load or timeout
+- Use of computer-vision capabilities now requires authentication
+- Sprite rendering within the SDK has been improved
+- Fixed a bug where the overlay would throw a null exception when opening for the first time
+- Fixed issues where the game would hang when starting
```
Comment (RG-nAmKcAz) [inline docs/changelog.mdx:26]:
```suggestion
- Fixed issues causing the game to freeze for a few seconds when starting
```

**Review by RG-nAmKcAz — CHANGES_REQUESTED** [review]

File: docs/changelog.mdx (hunk 1 above)
Comment (vontell) [reply docs/changelog.mdx]: Added

**Review by RG-nAmKcAz — APPROVED** [review]

**Comment by vontell** [comment]
@abeizer will wait for your review before merging

**Review by abeizer — APPROVED** [review]

**Pull request merged**